master/master.cfg

```diff
@@ -13,35 +13,37 @@
 import os
 import sys
 
+from datetime import timedelta
+
 from buildbot.schedulers.basic import SingleBranchScheduler
 from buildbot.schedulers.timed import Nightly
 from buildbot.schedulers.forcesched import (ForceScheduler, FixedParameter,
                                             StringParameter, UserNameParameter)
+from buildbot.plugins import reporters, util
 from buildbot.process import factory
-from buildbot.status import html, mail, words
 from buildbot.steps.shell import ShellCommand, Configure, Compile
 from buildbot.steps.transfer import DirectoryUpload
 from buildbot import locks
 from buildbot.steps.source.git import Git
+from buildbot.www.auth import RemoteUserAuth
```

Comment:
I'd rather leave this import in the `except ImportError` block of the 'local' setup.  Also, I'm not sure which auth we actually want.  `NoAuth` is probably closer to what we're accustomed to, but I'm interested in and have experimented a bit with `GitHubAuth` (and found that it isn't Python3 compatible).

Reply:
I removed RemoteUserAuth from that line as you requested.

Reply:
Looks like it's still here :)